test/stdlib/TestNSString.swift

```diff
@@ -30,16 +30,40 @@ import FoundationBridgeObjC
 
 class TestNSString : TestNSStringSuper {
     
-    func test_equalOverflow() {
-      let cyrillic = "чебурашка@ящик-с-апельсинами.рф"
-      let other = getNSStringEqualTestString()
-      print(NSStringBridgeTestEqual(cyrillic, other))
-    }
+  func test_equalOverflow() {
+    let cyrillic = "чебурашка@ящик-с-апельсинами.рф"
+    let other = getNSStringEqualTestString()
+    print(NSStringBridgeTestEqual(cyrillic, other))
+  }
+  
+  func test_smallString_BOM() {
+    let bom = "\u{FEFF}" // U+FEFF (ZERO WIDTH NO-BREAK SPACE)
+//    expectEqual(1, NSString(string: bom).length)
```

Comment:
These will continue to fail until we fix the non-smol issues, so are commented out for now

Reply:
You can also split out this portion and XFAIL it instead, with the JIRA linked.

```swift
  .xfail(.always("SR-nnnn: ..."))
```